core/calculator.js

```diff
@@ -332,13 +447,10 @@ function generatelist() {
 		}
 	}
 
-
-
-	generate_chart(resource_tracker, generation_totals);
+	generate_chart(resource_tracker, generation_totals, used_from_inventory);
```

Comment:
Is `used_from_inventory` necessary to pass in to `generate_chart()`?

Reply:
I use it in line 848 to get the output amount for the extra element for `[from Inventory]` items. I think it's more explicit if there is an extra column for those entries.

Reply:
See comment below about that line, however it is still used in the block right after that so this will be necessary for now and I am not going to hold up the pr to see if there is a nicer way to handle this.